homework/list.js

```diff
@@ -52,28 +44,43 @@
       buff.appendChild(li);
     });
     ul.appendChild(buff);
-    _wrapper.appendChild(ul);
-  }
+    this._wrapper.appendChild(ul);
+  },
 
-  function fetchList(afterFetch) {
+  fetchList(afterFetch) {
```

Comment:
You have successfully "Promisified" the method, so you should remove the callback argument as well.